community-operators/svt-app-operator/svt-app-operator.v0.0.8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,244 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: svt-app-operator.v0.0.8
+  namespace: placeholder
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: "OpenShift Optional"
+    certified: "false"
+    description: An operator to run the SVT app
+    containerImage: docker.io/hongkailiu/svt-app-operator:a012
+    createdAt: 2019-01-01T11:59:59Z
+    support: RH SVT
+spec:
+  version: 0.0.8
+  displayName: SVT Application
+  provider:
+    name: RH SVT
+    url: www.redhat.com
+  maintainers:
+    - email: svt@redhat.com
+      name: RH SVT
+  maturity: alpha
+  labels:
+    olm-owner-enterprise-app: svt-app-operator
+    olm-status-descriptors: svt-app-operator.v0.0.8
+  description: SVT Tool in GoLang
+  icon:
+    - base64data: |-
+        iVBORw0KGgoAAAANSUhEUgAAAIAAAACACAYAAADDPmHLAAAX7UlEQVR4nO2de2xT5/nHv+f4co4v
```

Comment:
Does this image render properly in OpenShift's Operator Hub UI? I'm not sure if multiline strings work.

Reply:
See my comment below.